js/photoeditor.js

```diff
@@ -77,6 +77,18 @@ var photoEditor = {
         document.getElementById('pe-undoBttn').onclick = this.undo.bind(this);
         document.getElementById('pe-redoBttn').onclick = this.redo.bind(this);
 
+        //fix by team wp
+        document.getElementById("pe-downloadBtn").addEventListener('click', function() {
+            tmp.ctx = document.getElementById("pe-panel").getContext("2d");
+            tmp.image.src = tmp.ctx.canvas.toDataURL();
```

Comment:
I'm not sure you can download a jpeg image by this way.
As far as I know, when downloading a canvas element in a format, we have to set that format in advance, for example:
var image = canvas.toDataURL ("image / jpg");

And I find that default downloading as jpeg is not good, because it reduces the quality of the image. Of course that is ok if the user wants to do it with a special purpose.

Reply:
so, i want to download the image in its original format

Reply:
yes, we got it.
we will make it again.